04_numeronym/mohitnag/main_test.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNumeronyms(t *testing.T) {
+	testData := []struct {
+		Scenario string
+		input    []string
+		expected []string
+	}{
+		{"Scenario One", []string{"ehi!@  45%!"}, []string{"e9!"}},
+		{"Scenario two", []string{""}, []string{""}},
+	}
+	for _, td := range testData {
+		td := td
+		t.Run(td.Scenario, func(t *testing.T) {
+			numeronym := numeronyms(td.input...)
+			assert.Equal(t, td.expected, numeronym)
+			fmt.Println(numeronym)
```

Comment:
Can you please remove this print statement, this is not required

Reply:
Done